docs/tutorial/nbwidgets.rst

```diff
@@ -135,36 +159,36 @@ Again, we can retrieve the selected data just as with :class:`~lumicks.pylake.Fd
     original.f.plot()
     sliced.f.plot(start=original.start)
 
-.. image::  figures/nbwidgets/fd_dist_widget2.png
-
-The returned F,d curves correspond to the longest contiguous (in time) stretch of data that falls
-within the distance thresholds. However, noise in the distance measurement can lead to short gaps of the time
-trace falling slightly outside of the thresholds, as illustrated below:
-
-.. image:: figures/nbwidgets/fd_dist_widget3a.png
+    plt.tight_layout()
 
-To avoid premature truncation caused by this noise, there is an additional `max_gap` keyword argument
-to :class:`~lumicks.pylake.FdDistanceRangeSelector` that can be used to adjust the acceptable length of noise gaps. The default values
-is zero, such that all data points are guaranteed to fall within the selected distance range. The effect of this
-argument is shown below for an F,d curve sliced with the same distance thresholds:
+.. image:: figures/nbwidgets/fd_dist_widget2.png
 
-.. image:: figures/nbwidgets/fd_dist_widget3.png
+The returned F,d curves correspond to the longest contiguous (in time) stretch of data that falls
+within the distance thresholds.
 
 Range selection of single curve
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The selector widgets can also be easily accessed from single F,d curve instances::
+The selector widgets can also be easily accessed from single :class:`~lumicks.pylake.fdcurve.FdCurve` instances using :meth:`~lumicks.pylake.fdcurve.FdCurve.range_selector`::
 
-    fdcurve = fdcurves["Fd pull #6"]
+    plt.figure()
+    fdcurve = fdcurves["40"]
     t_selector = fdcurve.range_selector()
-    d_selector = fdcurve.distance_range_selector(max_gap=3)
+
+.. image:: figures/nbwidgets/single_curve_widget1.png
+
+And for the distance-based selector we can use :meth:`~lumicks.pylake.fdcurve.FdCurve.distance_range_selector`::
+
+    d_selector = fdcurve.distance_range_selector()
+
+.. image:: figures/nbwidgets/single_curve_widget2.png
 
 .. _crop_and_rotate:
 
 Cropping and Rotating Image Stacks
 ----------------------------------
 
-You can interactively define the location of a tether for a :class:`~lumicks.pylake.ImageStack` by using::
+You can interactively define the location of a tether for a :class:`~lumicks.pylake.ImageStack` by using :meth:`~lumicks.pylake.ImageStack.crop_and_rotate`::
```

Comment:
Thanks for adding the return type to the docstring here; however the link to the class doesn't work. Just use the class name (not the RST :class: link)

Additionally could you copy/paste the `Actions` section from `crop_and_rotate` to `ImageEditorWidget` 🙏

Reply:
Unfortunately, it only works if you add it to the listing (which I just did).